hack/make/release-deb

```diff
@@ -122,7 +122,7 @@ apt-ftparchive clean "$APTDIR/conf/apt-ftparchive.conf"
 # run the apt-ftparchive commands so we can have pinning
 apt-ftparchive generate "$APTDIR/conf/apt-ftparchive.conf"
 
-for dir in contrib/builder/deb/*/; do
+for dir in contrib/builder/deb/${DOCKER_ENGINE_GOARCH}/*/; do
```

Comment:
ok so considering the debs will all be released from a amd64 machine we need a better way to do this. I am happy to make the patch in a different PR because I know what I have in mind